src/pages/CalculatorPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Calculator from '../components/Calculator';
+import '../style/CalculatorPage.css';
+
+const CalculatorPage = () => (
+  <section className=" container calculator-section">
+    <h2> Let&#39;s do some math ! </h2>
+    <div className="calculator">
+      <Calculator />
+    </div>
+  </section>
+);
```

Comment:
- [ ] I suggest that you add a color that makes it easier to see numbers on your calculator this is how it looks like now:
![cal](https://user-images.githubusercontent.com/44745855/159548892-4a92edaf-ca5a-4a26-bc5a-346b8d1cc1f2.png)